app/src/main/java/com/github/se/signify/ui/screens/home/ExerciseScreen.kt

```diff
@@ -65,7 +65,7 @@ import com.github.se.signify.ui.common.CameraBox
 @Composable
 fun ExerciseScreen(
     navigationActions: NavigationActions,
-    handLandMarkViewModel: HandLandMarkViewModel,
+    handLandMarkViewModel: HandLandmarkViewModel,
```

Comment:
```suggestion
    handLandmarkViewModel: HandLandmarkViewModel,
```
Following the logic you used to rename those variables, this one should be named like this, right?
It's a long file, I will let you make the changes

Reply:
I didn't bother changing these variable names, since the coaches told us not to refactor old files.
Fixed it anyway.